terraform/cloudsql/mysql/bind/provider.tf

```diff
@@ -1,9 +1,3 @@
-provider "mysql" {
```

Comment:
question: I know we don't need to keep compatibility backwards, but we won't be able to remove old binding if we delete this providers. Is this what we want?

Reply:
I believe that the instances created with previous versions will no longer be manageable anyway, and it seems that now is the best time to remove the provider. Keeping a dangling provider for years supporting a pre-GA version of the service feels illogical to me.

Reply:
Thanks for the answer. It makes sense to me.